### ISSUE TITLE
Version control in Rstudio

### ISSUE THREAD

**Issue by bradduthie** [StirlingCodingClub/version_control#2]
If anyone fancies writing up a guide to using git in Rstudio (@jmcvw ?), it would be great to add this as a section to the [version control notes](https://github.com/StirlingCodingClub/version_control/blob/master/vc_notes.Rmd) that explain how to use GitKraken and the command line interface.

**Comment by jmcvw** [comment]
Yep, I can do this @bradduthie, how long it takes just depends on my work : procrastination ratio. Also, I've never worked on a repo that others are working on, so what's the best way to do it?
Do I just clone straight into this?
Or should I set up my own separate repo, and somehow merge them later?
Will I have to make pull requests to update?

**Comment by bradduthie** [comment]
Thanks @jmcvw -- you should be able to just clone and pull from the repository, then push your changes (I did this when adding [a file](https://github.com/StirlingCodingClub/tips-tricks-n-tuts/blob/master/docs/pointers-in-c-for-scientists.md) to the [repository](https://github.com/StirlingCodingClub/tips-tricks-n-tuts) that @rozeykex initialised). It might be best to clone, then start a new branch to push and later merge.

**Comment by jmcvw** [comment]
Hi all,

I have just added RStudio_and_git.Rmd to the repo about using git from RStudio. So far it only really covers the different status a file can have, adding and commiting files, and how modifications are dealt with.

RStudio also makes it quite easy to deal with branching and merging, and I'll try and get round to those topics eventually.

**Comment by bradduthie** [comment]
Awesome! Thanks @jmcvw for adding the [RStudio_and_git.Rmd](https://github.com/StirlingCodingClub/version_control/blob/master/RStudio_and_git.Rmd) tutorial. I still need to try to this out in Rstudio.
